operator/docs/examples/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+kind: StorageClass
```

Comment:
Thoughts on using a default storage class so we don't need to specify in most cases? 
https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource

Reply:
You are 100% right. We definitely need a default storageclass. So if no storage class is defined in the controlplane spec, the default will be used. We need to specify a high IOPS for the default (currently 100 IOPS is like a drop in the bucket).

So to continue the discussion, I was thinking that if we use AWS-EBS-CSI driver, KIT should be able to create different storageclass for different tests so that we can also scale test the impact of storage (e.g., IOPS, size, type). This should be optional in the contorlplane spec.